src/index.css

```diff
@@ -66,3 +66,18 @@ button:focus-visible {
     background-color: #f9f9f9;
   }
 }
+
+ul {
+  list-style: none;
+}
+
+header {
+  background-color: white;
```

Comment:
since you're still working with the Vite & React CSS template, I can't see the header on dark mode on my computer because the background is white and the text colour is white